R/bq-stream.R

```diff
@@ -0,0 +1,89 @@
+#' Streams data from data.from into BigQUery
```

Comment:
```suggestion
#' Stream data.frame into into BigQUery
```

Can you please also add a sentence or two about why you'd use this function? (i.e. what you have in the description of this PR)